provider/datagen/src/transform/cldr/datetime/symbols.rs

```diff
@@ -60,10 +60,28 @@ fn get_month_code_map(calendar: &str) -> &'static [TinyStr4] {
         tinystr!(4, "M13"),
     ];
 
+    static HEBREW_MONTH_CODES: &[TinyStr4] = &[
+        tinystr!(4, "M01"),
+        tinystr!(4, "M02"),
+        tinystr!(4, "M03"),
+        tinystr!(4, "M04"),
+        tinystr!(4, "M05"), 
+        tinystr!(4, "M06"),
+        tinystr!(4, "M07"),
+        tinystr!(4, "M08"),
+        tinystr!(4, "M09"),
+        tinystr!(4, "M10"),
+        tinystr!(4, "M11"),
+        tinystr!(4, "M12"),
+        tinystr!(4, "M05L"),
+        tinystr!(4, "M06L"),
+    ];
+
     match calendar {
         "gregory" | "buddhist" | "japanese" | "japanext" | "indian" | "persian" | "roc"
         | "islamic" | "islamicc" | "umalqura" | "tbla" => &SOLAR_MONTH_CODES[0..12],
         "coptic" | "ethiopic" | "chinese" => SOLAR_MONTH_CODES,
+        "hebrew" => HEBREW_MONTH_CODES,
```

Comment:
we should just `panic!("Should not reach this branch for hebrew")`, OR use this month code map below.

My preference is that we have this month code map in sorted order for CLDR (so 05L comes after 05), and in the code below we have a special case for _just_ `7-yeartype-leap`

Reply:
Okay, yeah, since `HEBREW_MONTH_CODES` is currently unused, we should delete it. So just delete this line, delete the variable, and change line 14 to pass in an empty list.

Reply:
I really think we should go back to my original suggestion of having HEBREW_MONTH_CODES in this function and specialcasing a _little bit_ in the function below but largely relyign on the month codes map

currently this errors because now this function is called without a match arm